packages/config/config/devices/0x0212/gda-a2e6b-k0.json

```diff
@@ -0,0 +1,88 @@
+// GATEMAN GDA-A2E6B-K0
+// A20-SH
+{
+	"manufacturer": "GATEMAN",
+	"manufacturerId": "0x0212",
+	"label": "GDA-A2E6B-K0",
+	"description": "A20-SH",
+	"devices": [
+		{
+			"productType": "0x1000",
+			"productId": "0x1000",
+			"zwaveAllianceId": 1434
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Silent Mode On/Off",
+			"description": "",
```

Comment:
remove

Reply:
You mean remove empty descriptions?

Reply:
Yep. See https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=paraminformation